### DISCUSSION TITLE
Latex  rendering

### DISCUSSION THREAD

**Discussion by Amir1453** [epwalsh/obsidian.nvim#519] [Ideas]
Is latex rendering (similar to markdown tasks for example) planned/on the roadmap?

**Comment by luxShrine** [comment]
Since this seems to be a niche issue that I don't see getting a good feature implemented, I have setup vimtex for .md files and just use the conceal functionality for small snippets of latex. And mainly use the obsidian's app's Latex Suite's rendering of my tex for really visualizing things. 

Thus, my workflow kinda becomes a simpler form of compiling and viewing a typical tex file. I navigate to a note in nvim, and I write some latex with snippets, write it, and open the app to view the pretty latex. I would of course prefer a more streamlined approach like how seamless latex is integrated natively in the app with latex-suite. But, it isn't awful as is. Attached is a short video of what it looks like https://github.com/user-attachments/assets/3d66f4cc-7d22-4f49-a7c7-36e308ae01da
